spec/definitions.yaml

```diff
@@ -732,7 +732,7 @@ PatchOperation:
     - path
     - value
 CustomProperties:
-  type: object
+  type: array
```

Comment:
relates to https://github.com/mlafeldt/go-launchdarkly/pull/2/files#diff-bac86ae86657c2fa2c3dd2ab26df4960R1610